packages/embed/src/factories/create-popover/create-popover.ts

```diff
@@ -133,11 +177,30 @@ export const createPopover = (formId: string, userOptions: PopoverOptions = {}):
     }
   }
 
+  const hideNotificationDot = () => {
+    if (notificationDot) {
+      notificationDot.classList.add('closing')
+      setTimeout(() => {
+        notificationDot.remove()
+      }, 250)
+    }
+  }
+
   if (options.tooltip && options.tooltip.length > 0) {
     tooltip = buildTooltip(options.tooltip, hideTooltip)
     popover.append(tooltip)
   }
 
+  if (options.chat && options.notificationDot) {
```

Comment:
Can we enable this for all popovers? We will not expose it in share page, but get it working "for free" for all embeds when people integrate via SDK which I think is nice. Tooltip works form all embeds too.

Reply:
Just got it clarified by Max, I thought it was only for chat